packages/pageflow-react/src/loadingSpinner/components/MediaLoadingSpinner.jsx

```diff
@@ -82,13 +82,22 @@ function preventScrollBouncing(e) {
   e.preventDefault();
 }
 
-function backgroundImageInlineStyles({firstPageBackgroundImageUrl, backgroundImage, blurStrength}) {
+function backgroundImageInlineStyles({firstPageBackgroundImageUrl, backgroundImage, blurStrength, backgroundImageX, backgroundImageY}) {
+  
+  var backgroundPosition = {
+    x: backgroundImageX ? backgroundImageX : 50,
+    y: backgroundImageY ? backgroundImageY : 50
```

Comment:
Since 0 is falsy, this makes it fall back to 50 when 0 is selected as position.

Reply:
oops